etc/kayobe/kolla/config/prometheus/extras/file_sd/ipmi-exporter-targets.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This is a question really... not sure about why this is here, I guess it was generated by the others scripts?

Reply:
Yep, these files were generated. We could probably move the templating here instead. Do you think that would be cleaner?